pkgs/top-level/python-packages.nix

```diff
@@ -6452,6 +6452,10 @@ in {
 
   salmon-mail = callPackage ../development/python-modules/salmon-mail { };
 
+  sane = disabledIf (!isPy3k) (callPackage ../development/python-modules/sane {
```

Comment:
Looks like it is still more common to disable packages for Python 2 in the expressions themselves:

```ShellSession
$ rg 'disabledIf' pkgs/top-level/python-packages.nix | wc -l
34
$ rg -l 'disabled =' pkgs/development/python-modules/ | wc -l
1016
```

Reply:
Since python2 is no longer built by default, I removed the disabledIf totally.